src/sharedInterfaces/queryResult.ts

```diff
@@ -11,11 +18,43 @@ export enum QueryResultLoadState {
 
 export interface QueryResultReactProvider {
     setResultTab: (tabId: QueryResultPaneTabs) => void;
+    /**
```

Comment:
Maybe extract it out to a parent interface shared between the 2 webviews.
(execution plan and query result)